src/sidebar/app-controller.js

```diff
@@ -25,7 +27,7 @@ function authStateFromUserID(userid) {
 module.exports = function AppController(
   $document, $location, $rootScope, $route, $scope,
   $window, annotationUI, auth, drafts, features, frameSync, groups,
-  serviceUrl, session, settings, streamer
+  serviceUrl, session, settings, streamer, bridge
```

Comment:
The service list short be sorted. Otherwise it is difficult to see whether a particular service is already in the list at a glance.